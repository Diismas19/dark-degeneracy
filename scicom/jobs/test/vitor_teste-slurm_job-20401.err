/home/valerio/anaconda3/envs/vitor/lib/python3.12/site-packages/act_dr6_lenslike/act_dr6_lenslike.py:422: UserWarning: Hartlap correction to cinv: 0.9860935524652339
  warnings.warn(f"Hartlap correction to cinv: {hartlap_correction}")
slurmstepd: error: *** JOB 20401 ON node9 CANCELLED AT 2025-04-15T09:11:26 ***
