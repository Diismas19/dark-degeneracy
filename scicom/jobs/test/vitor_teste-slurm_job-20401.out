/home/valerio/Vitor/jobs/test
[output] Output to be read-from/written-into folder 'chains', with prefix 'int'
[prior] *WARNING* External prior 'SZ' loaded. Mind that it might not be normalized!
[classy] `classy` module loaded successfully from /home/valerio/Vitor/cobaya/class/build/lib.linux-x86_64-cpython-312
Loading ACT DR6 lensing likelihood v1.2...
[bao.desi_dr2] Initialized.
[planck_2018_highl_plik.ttteee] `clik` module loaded successfully from /home/valerio/Vitor/cobaya/code/planck/clik-main/lib/python/site-packages/clik
[mcmc] Getting initial point... (this may take a few seconds)
[mcmc] Initial point: logA:3.050126, n_s:0.9724956, theta_s_100:1.041316, omega_b:0.02239539, omega_cdm:0.1201954, w0_int:-1.009409, wa_int:-0.02409539, tau_reio:0.04821665, A_planck:1.000855, calib_100T:1.000132, calib_217T:0.9989094, A_cib_217:76.83422, xi_sz_cib:0.0766937, A_sz:4.954918, ksz_norm:3.85366, gal545_A_100:7.288828, gal545_A_143:5.371752, gal545_A_143_217:21.85986, gal545_A_217:70.05813, ps_A_100_100:237.2242, ps_A_143_143:43.93067, ps_A_143_217:44.78731, ps_A_217_217:94.06432, galf_TE_A_100:0.09971618, galf_TE_A_100_143:0.09060794, galf_TE_A_100_217:0.4823082, galf_TE_A_143:0.2691747, galf_TE_A_143_217:0.8290131, galf_TE_A_217:1.744981
[model] Measuring speeds... (this may take a few seconds)
[model] Setting measured speeds (per sec): {act_dr6_lenslike.ACTDR6LensLike: 20.5, sn.desy5: 933.0, bao.desi_dr2: 2480.0, planck_2018_lowl.TT: 2180.0, planck_2018_lowl.EE: 5660.0, planck_2018_highl_plik.TTTEEE: 73.8, planckpr4lensing: 2440.0, classy: 0.107}
[mcmc] Dragging with number of interpolating steps:
[mcmc] *  1 : [['logA', 'n_s', 'theta_s_100', 'omega_b', 'omega_cdm', 'w0_int', 'wa_int', 'tau_reio']]
[mcmc] * 32 : [['A_planck'], ['calib_100T', 'calib_217T', 'A_cib_217', 'xi_sz_cib', 'A_sz', 'ksz_norm', 'gal545_A_100', 'gal545_A_143', 'gal545_A_143_217', 'gal545_A_217', 'ps_A_100_100', 'ps_A_143_143', 'ps_A_143_217', 'ps_A_217_217', 'galf_TE_A_100', 'galf_TE_A_100_143', 'galf_TE_A_100_217', 'galf_TE_A_143', 'galf_TE_A_143_217', 'galf_TE_A_217']]
[autoselect_covmat] No cached covmat database present, not usable or not up-to-date. Will be re-created and cached.
[mcmc] Covariance matrix selected automatically: /home/valerio/Vitor/cobaya/data/planck_supp_data_and_covmats/covmats/base_plikHM_TTTEEE_lowl_lowE_lensing_post_BAO.covmat
[mcmc] Covariance matrix loaded for params ['omega_b', 'omega_cdm', 'tau_reio', 'logA', 'n_s', 'A_planck', 'A_cib_217', 'xi_sz_cib', 'A_sz', 'ps_A_100_100', 'ps_A_143_143', 'ps_A_143_217', 'ps_A_217_217', 'ksz_norm', 'gal545_A_100', 'gal545_A_143', 'gal545_A_143_217', 'gal545_A_217', 'galf_TE_A_100', 'galf_TE_A_100_143', 'galf_TE_A_100_217', 'galf_TE_A_143', 'galf_TE_A_143_217', 'galf_TE_A_217', 'calib_100T', 'calib_217T']
[mcmc] Missing proposal covariance for params ['theta_s_100', 'w0_int', 'wa_int']
[mcmc] Covariance matrix not complete. We will start learning the covariance of the proposal earlier: R-1 = 30 (would be 2 if all params loaded).
[mcmc] Sampling!
[mcmc] Progress @ 2025-04-15 09:10:57 : 1 steps taken, and 0 accepted.
